components/post_view/reaction/reaction.test.tsx

```diff
@@ -19,7 +19,7 @@ describe('components/post_view/Reaction', () => {
     };
     const currentUserId = 'user_id_1';
 
-    const baseProps = {
+    const baseProps: any = {
```

Comment:
Which props are missing from baseProps? We generally want to add dummy values for any unused props since we'll get a warning then whenever something new is added